@here/harp-materials/lib/ShadowFeature.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2020 HERE Europe B.V.
+ * Licensed under Apache 2.0, see full license in LICENSE
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { chainCallbacks } from "@here/harp-utils";
+import * as THREE from "three";
+import { HiddenThreeJSMaterialProperties } from "./MapMeshMaterials";
+import { simpleLightingShadowChunk } from "./ShaderChunks/ShadowChunks";
+
+/**
+ * Parameter used to control patching the THREE.MeshStandardMaterial shader to ensure that the
+ * material's color isn't affected by the light direction.
+ */
+export interface ShadowFeatureParameters {
```

Comment:
W.r.t to removing "ShadingFeature"  i think i meand .... completly :)
I think now this interface is not needed at all.

All we need is just add this `removeDiffuseLight` to MapMeshMaterialParameters which should be named and look more or less like this:

```
interface MapMeshMaterialParameters extends THREE...., FirstMixin, SecondMixin, ... {
    removeDiffuseLight?: ...
}
```

Just to save some bytes and names.

Reply:
Ok, will do.

Reply:
As discussed, will remove in favor of the approach of using the & { removeDiffuseLight }